charts/kusion/README.md

```diff
@@ -46,9 +52,9 @@ Edit the [values file](values.yaml) to add your kubeconfig content
 # Replace the content below with your actual kubeconfig
 kubeconfig:
```

Comment:
The default kubeconfig path should be documented more clearly to indicate it's a Kubernetes standard path, not just an arbitrary choice